relying_party.go

```diff
@@ -148,7 +148,7 @@ func (rp *RelyingParty) VerifyAuthenticationCeremony(
 	}
 
 	// 13. Verify that the value of C.origin matches the Relying Party's origin.
-	if !stringsAreEqual(clientData.Origin, rp.origin) {
+	if !originMatches(clientData.Origin, rp.origin) {
 		return nil, fmt.Errorf("invalid client data origin")
```

Comment:
Should we bubble up the rp / client origins provided in case we need to troubleshoot?